OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These changes should not be committed (same with the changes at the end of the file).